project.clj

```diff
@@ -55,15 +55,15 @@
                                     :password      :env/clojars_password
                                     :sign-releases false}]]
   :pedantic? :warn
-  :java-source-paths ["src/com"]
+  :java-source-paths ["src/com" "test/com"]
   :aliases {"test-all"      ["with-profile" "default:+1.8:+1.9" "test"]
             "code-coverage" ["with-profile" "test" "cloverage" "--output" "coverage" "--coveralls"]}
   :aot [ziggurat.init ziggurat.config ziggurat.producer ziggurat.sentry ziggurat.metrics ziggurat.fixtures]
   :profiles {:uberjar {:aot         :all
                        :global-vars {*warn-on-reflection* true}
                        :pedantic?   :abort}
              :test    {:jvm-opts     ["-Dlog4j.configurationFile=resources/log4j2.test.xml"]
-                       :dependencies [[com.google.protobuf/protobuf-java "3.9.1"]
+                       :dependencies [[com.google.protobuf/protobuf-java "3.5.1"]
```

Comment:
Why downgrade this version?

Reply:
`clojusc/protobuf` version we are using is 3.5.1. In order to keep it consistent across, I have downgraded the version. If there is a version mismatch, the custom proto file that I have generated to test for the feature will fail to compile.